third-party/linshare/src/test/java/org/apache/james/linshare/Linshare.java

```diff
@@ -115,7 +121,7 @@ private GenericContainer createDockerBackend() {
             new ImageFromDockerfile()
                 .withFileFromClasspath("conf/log4j.properties", "backend/conf/log4j.properties")
                 .withFileFromClasspath("conf/catalina.properties", "backend/conf/catalina.properties")
-                .withFileFromClasspath("conf/id_rsa", "backend/conf/id_rsa")
+                .withFileFromClasspath("conf/id_rsa", "backend/conf/id_rsa.pri")
```

Comment:
does this change pass Linshare Client tests?

Reply:
It doesn't, as the bind in the dockerfile remains the same at the end. That's the only way I found to go around the weird error with `LinshareBlobExportMechanismProvidingTest` that I posted a bit above